File: uva/c_cpp/uva-12289.cpp
#include <cstdio>
#include <iostream>
#include <string>
#include <cstring>

using namespace std;

int main()
{
    string input;
    int len,i,cas,T;

    cin >> T;

    while(T--)
    {
        cin >> input;

        len = input.size();

        if( len > 3 ) cout << "3\n";
        else if( (input[0] == 't' && input[1] == 'w') || (input[0] == 't' && input[2] == 'o') || (input[2] == 'o' && input[1] == 'w')  )
        {
            cout << "2\n";
        }
        else
            cout << "1\n";
    }
    return 0;
}

File: uva/java/uva_12289/src/uva_12289/Uva_12289.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package uva_12289;

import java.util.Scanner;

/**
 *
 * @author DELL
 */
public class Uva_12289 {

    /**
     * @param args the command line arguments
     */
    static Scanner scan = new Scanner(System.in);

    public static void main(String[] args) {
        // TODO code application logic here

        String input;
        int len, i, cas, T;

        //cin >> T;
        T = scan.nextInt();
        scan.nextLine();

        while (T-- > 0) {
            //cin >> input;
            
            input = scan.nextLine();

            len = input.length();

            if (len > 3) {
                System.out.println( "3");
            } else if ((input.charAt(0) == 't' && input.charAt(1) == 'w') || (input.charAt(0) == 't' && input.charAt(2) == 'o') || (input.charAt(2) == 'o' && input.charAt(1) == 'w')) {
                System.out.println("2");
            } else {
                System.out.println("1");
            }
        }
    }

}
